app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -34,11 +40,11 @@ class ProfileEditActivityPresenter @Inject constructor(
       activity,
       R.layout.profile_edit_activity
     )
-    val profileId = activity.intent.getIntExtra(PROFILE_EDIT_PROFILE_ID_EXTRA_KEY, 0)
-    profileEditViewModel.setProfileId(profileId)
+    profileId = activity.intent.getIntExtra(PROFILE_EDIT_PROFILE_ID_EXTRA_KEY, 0)
```

Comment:
Suggested:
`val profileId = activity.intent.getIntExtra(PROFILE_EDIT_PROFILE_ID_EXTRA_KEY, 0)`

Reply:
Surely will do this.